toolkit/tools/imagecustomizerapi/systemconfig_test.go

```diff
@@ -53,3 +53,41 @@ func TestSystemConfigIsValidKernelCommandLineInvalidChars(t *testing.T) {
 	assert.Error(t, err)
 	assert.ErrorContains(t, err, "ExtraCommandLine")
 }
+
+func TestSystemConfigIsValidVerityInvalidPartLabel(t *testing.T) {
+	invalidVerity := SystemConfig{
+		Verity: &Verity{
+			DataPartition: VerityPartition{
+				IdType: "PartUuid",
+				Id:     "0f2884c0-8fe0-4a19-87bf-286b7fe9d6f2",
+			},
+			HashPartition: VerityPartition{
+				IdType: "PartLabel",
+				Id:     "",
+			},
+		},
+	}
+
+	err := invalidVerity.IsValid()
+	assert.Error(t, err)
+	assert.ErrorContains(t, err, "invalid Id: empty string")
+}
+
+func TestSystemConfigIsValidVerityInValidPartUuid(t *testing.T) {
```

Comment:
You probably only need a single test in `systemconfig_test.go` to ensure that the `s.Verity.IsValid()` error check is covered. You can probably leave more specific tests to other files.

Reply:
Fixed. Thanks.